src/antsibull/constants.py

```diff
@@ -25,3 +25,25 @@
 ANSIBLE_FORUM_URL = "https://forum.ansible.com"
 
 COLLECTION_EXCLUDE_DIRS = ("docs", "tests")
+
+SANITY_TESTS_DEFAULT: tuple[str, ...] = (
+    "ansible-doc",
+    "compile",
+    "validate-modules",
+    "yamllint",
+)
+SANITY_TESTS_BANNED_IGNORES = frozenset(
+    {
+        "validate-modules!skip",
+        "validate-modules:doc-choices-do-not-match-spec",
+        "validate-modules:doc-default-does-not-match-spec",
+        "validate-modules:doc-missing-type",
+        "validate-modules:doc-required-mismatch",
+        "validate-modules:mutually_exclusive-unknown",
+        "validate-modules:no-log-needed",
+        "validate-modules:nonexistent-parameter-documented",
+        "validate-modules:parameter-list-no-elements",
+        "validate-modules:parameter-type-not-in-doc",
+        "validate-modules:undocumented-parameter",
```

Comment:
This one should be commented out, with the comment you previously had for the other one.
```suggestion
        # "validate-modules:undocumented-parameter",
```